itests/hive-unit/src/main/java/org/apache/hive/jdbc/miniHS2/MiniHS2.java

```diff
@@ -222,6 +222,8 @@ private MiniHS2(HiveConf hiveConf, MiniClusterType miniClusterType, boolean useM
       // Initialize the execution engine based on cluster type
       switch (miniClusterType) {
       case TEZ:
+        // Change the engine to tez
+        hiveConf.setVar(ConfVars.HIVE_EXECUTION_ENGINE, "tez");
         // TODO: This should be making use of confDir to load configs setup for Tez, etc.
```

Comment:
In the interest of being more explicit, should we set the "hive.execution.mode" to "container" as well, since LLAP setup has other steps.